src/ocaml/preprocess/411/lexer_raw.mll

```diff
@@ -140,6 +140,10 @@ let keyword_table : keywords =
     "asr", INFIXOP4("asr");
 ]
 
+let ocaml_keywords =
+  let l = lazy (Hashtbl.fold (fun k _ acc -> k :: acc) keyword_table []) in
+  fun () -> Lazy.force l
```

Comment:
Did you notice an impact on startup time if you remove the layer of lazy + closure? That is: if you just compute the list of keywords during module initialization.
There aren't that many of them, so hopefully it should be fine.

Reply:
As you mentioned, it wouldn't work without custom keywords. Although it makes sense to remove custom keywords altogether? I don't think anyone uses camlp4 anymore, so I'm not sure what the functionality is useful for .